### ISSUE TITLE
How not to download ideaIC-LATEST-EAP-SNAPSHOT.zip？

### ISSUE THREAD

**Issue by zxlc** [JetBrains/gradle-intellij-plugin#323]
I have configured ideaIC version '2018.1.4'  and the version ideaIC has been downloaded. 

But when I build my project, it alwayls downloads ideaIC-LATEST-EAP-SNAPSHOT.zip. Why? 
How to make it not download ideaIC-LATEST-EAP-SNAPSHOT.zip？

Here is my build.gradle:
```
plugins {
    id 'java'
    id 'org.jetbrains.intellij' version '0.3.11'
}

group 'com.example.ideaplugin'
version '1.0.0-SNAPSHOT'

sourceCompatibility = 1.8

repositories {
    mavenCentral()
}

dependencies {
    compile ""
    testCompile group: 'junit', name: 'junit', version: '4.12'
}

intellij {
    version '2018.1.4'
}

patchPluginXml {
    changeNotes """
      Add change notes here.<br>
      <em>most HTML tags may be used</em>"""
}
```

**Comment by zolotov** [comment]
How do you build your project?

**Comment by zxlc** [comment]
I just created an idea plugin project based on gradle, and change the ideaIC version from 2018.1.5 to 2018.1.4.  It is an empty project

**Comment by zolotov** [comment]
> I just created an idea plugin project based on gradle, and change the ideaIC version from 2018.1.5 to 2018.1.4. It is an empty project

I cannot recreate this. And that snippet doesn't look like a template:

```
dependencies {
    compile ""
    testCompile group: 'junit', name: 'junit', version: '4.12'
}
```

So how do build the project? Can you attach building log when it downloads LATEST-SNAPSHOT?

**Comment by zolotov** [comment]
Actually looks like `compile ""` is the reason. By that string you broke the script so the version had never been passed to gradle-intellij-plugin, default one is used

**Comment by enbandari** [comment]
I came up with the same problem and fixed by upgrading gradle version to 4.8.1. Maybe helps.
